app/Http/Controllers/AuthController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Carbon\Carbon;
+use App\User;
+
+/*
+    TODO: Better error handling (e.g. what if createToken fails etc.).
```

Comment:
comment block format:
/*
 *
*/
or something like that

Reply:
That format breaks my editor plugin which highlight "TODO" etc, comments and lets me search for them and view them all in one place... :unamused:

Reply:
lets ignore this then